docs/cross-platform-targeting.md

```diff
@@ -64,6 +35,48 @@ Targeting .NET Standard, and successfully compiling your project, does not guara
 * [.NET Standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
 * [.NET API analyzer](https://docs.microsoft.com/en-us/dotnet/standard/analyzers/api-analyzer)
 
+## Multi-targeting
+
+An alternative way to add cross-platform support to a .NET library is multi-targeting. Multi-targeting involves targeting .NET implementations individually and includes multiple assemblies in a NuGet package. This leverages NuGet's ability to have multiple assemblies in a package and select the best one when added to an application.
+
+You can combine multi-targeting with .NET Standard. For example, a `netstandard2.0` target could be the default implementation and you could provide a .NET implementation specific target that uses implementation specific APIs for additional features. NuGet will automatically select the implementation specific target when possible, e.g. a .NET Core application will use the `netcoreapp2.0` assembly over the `netstandard2.0` assembly in a NuGet package.
+
```

Comment:
I don't think we should position multi-targeting as an *alternative* to .NET Standard. While it *could* be used this way, that's not what we recommend because it forwards the multi-targeting requirement to the consumer. Rather, our recommendation is to build bait-and-switch-style .NET Standard libraries where you have always target .NET Standard while *also* providing framework-specific implementations.

Hence, I suggest we reword this to something like this:

> Sometimes you need to access framework-specific APIs from your libraries. The best way to do this is by using *multi-targeting*, which builds your project  for multiple frameworks rather than for just one.
>
> In order to shield your consumers from having to build for multiple frameworks too you should always produce a .NET Standard output plus one or more framework-specific outputs. All outputs will be packaged inside a single NuGet package, allowing consumers to reference the same package and let NuGet pick the appropriate implementation. Your .NET Standard library serves as the fall-back library that is used everywhere, except for the cases where your NuGet package offers a framework-specific implementation. This enables you to use conditional compilation in your code in order to call framework-specific APIs.